documentation/src/docs/contributing.md

```diff
@@ -0,0 +1,7 @@
+# Contributing to Pravega Operator
+
+Please check here for [Contributions guidelines](https://github.com/pravega/pravega-operator/wiki/Contributing)
```

Comment:
Replace with:

Please check the [Contributing](https://github.com/pravega/pravega-operator/wiki/Contributing) wiki page.

Reply:
sure will edit